sycl/test/native_cpu/vector-add.cpp

```diff
@@ -0,0 +1,51 @@
+// REQUIRES: native_cpu_be
+// REQUIRES: native_cpu_be
```

Comment:
```suggestion
// REQUIRES: native_cpu_be
```

Could you please also explain why compilation process is so complex for this test comparing to others?

Reply:
There is no particular reason, it's mainly because I added this test before having support to the `-fsycl` flag with `native_cpu` in the Driver, I'm happy to change it to the same compiler invocation used in the other tests if you think it's more appropriate. This test right now is also checking that performing the two-step compilation manually works for Native CPU (which allows users to use an arbitrary host compiler), but I understand that this will become less of a priority with the 1-pass compiler.